tests/test_api_v1.py

```diff
@@ -515,6 +515,31 @@ def test_extra_query_parameter(client):
     assert rv.json["error"] == "Extra query parameter(s) tom_hanks not in spec"
 
 
+@pytest.mark.parametrize(
+    "date, is_valid, expected_status",
+    [
+        ("tom_hanks", False, 400),
+        ("2011-11-04", True, 200),
+        ("2011-11-04T00:05:23", True, 200),
+        ("2011-11-04 00:05:23.283", True, 200),
```

Comment:
We could also add some more invalid scenarios such as:
`2011-20-01` (invalid month)
`2011-01-01T24:00:00` (invalid time)
`2011-01-01T22:00:00.0` (invalid miliseconds)